### PR TITLE
Feature/add ngx socket

### PR DESCRIPTION
**Pull request by MariaRoz**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -17,6 +17,11 @@ import { EffectsModule } from '@ngrx/effects';
 import { AuthModule } from './modules/auth/auth.module';
 import { AppLoadService } from './app-load.service';
 import { initializeApp } from './initialize-app';
+import { AppEffects } from './store/app.effects';
+import { SocketIoConfig, SocketIoModule } from 'ngx-socket-io';
+
+const config: SocketIoConfig = { url: 'http://localhost:3000', options: {}};
```
Comment (myownsumm) [inline src/app/app.module.ts]: use Angular configuration feature providing these settings

File: src/app/store/app.effects.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+@Injectable()
+export class AppEffects {
+
+  constructor(private socket: Socket) {}
+
+  @Effect()
+  takeSocketEvent$ = this.socket.fromEvent('chat').pipe(
+    map((res: {action: string, payload?: object}) => {
+      return new MessageActions.ChatUpdated(res);
```
Comment (myownsumm) [inline src/app/store/app.effects.ts]: Fix your WS actions structure. It should be `{type: string, payload?: object}`
Do not dispatch this specific action here. Use:
`of({ type, payload }))`

Comment (MariaRoz) [reply src/app/store/app.effects.ts]: @myownsumm Do you mean something like this:
Screen Shot 2019-11-27 at 3 16 34 PM

Comment (MariaRoz) [reply src/app/store/app.effects.ts]: Fixed

**Pull request merged**